### PR TITLE
express, dotenv dependencies added, nodemon devDependency added, setu…

### PR DESCRIPTION
**Pull request by harjothkhara**
…p server and start scripts. server, actionRouter, projectRouter files added, routes imported to server, server imported to index and initialized

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -26,14 +26,26 @@ Demonstrate your understanding of this Sprint's concepts by answering the follow
...
+
 - [ ] What can the API return to help clients know if a request was successful?
 
+http status codes (e.g res.status(200) for success), error handlers to catch and let the client know of unsucessful requests. we can add json messages as well to be clear to the client about what went wrong.
+
+
 - [ ] How can we partition our application into sub-applications?
 
+ we can use routers to split up our app into sub--applications and make incorporate modularity  - server, routes, routers, db ...organization allow for better readibility for the next reviewer of your code, and is easier to maintain.
+
```
Comment (angel-torres) [inline README.md]: Great responses! ✔️

File: actionRouter.js [hunk 2]
```diff
@@ -0,0 +1,109 @@
...
+      }
+    } catch (error) {
+      res
+        .status(500)
+        .json({ message: "Something went wrong when you made your request." });
+    }
+  }
+});
+
+module.exports = router;
```
Comment (angel-torres) [inline actionRouter.js]: ✔️

File: projectRouter.js [hunk 3]
```diff
@@ -0,0 +1,104 @@
...
+          .json({ message: "The post information could not be modified." });
+      }
+    } catch (error) {
+      res
+        .status(500)
+        .json({ message: "Something went wrong when you made your request." });
+    }
+  }
+});
+
```
Comment (angel-torres) [inline projectRouter.js]: Endpoints look good! ✔️

File: server.js [hunk 4]
```diff
@@ -0,0 +1,21 @@
+const express = require('express');
+const helmet = require("helmet");
+
+const projectRouter = require("./projectRouter.js");
+const actionRouter = require("./actionRouter.js");
+
+const server = express();
```
Comment (angel-torres) [inline server.js]: You could try using `cors` here.

**Review by angeltorgon — COMMENTED** [review]
# Project Manager Feedback
## Great
* You were able to complete all MVP objectives! :heavy_check_mark:
* You were able to set up different routes and use middleware in your express app! :heavy_check_mark:
Great work Harjoth!
## Requested Improvements
* Try making a `React` app to connect to your `express` app!
## Questions
* How comfortable to you feel setting up an express app?
## Rating {2}
:star: :star:

**Pull request merged**